awx/main/utils/formatters.py

```diff
@@ -107,6 +107,17 @@ def serialize(cls, message):
 
 class LogstashFormatter(LogstashFormatterBase):
 
+    def __init__(self, *args, **kwargs):
+        self.cluster_host_id = settings.CLUSTER_HOST_ID
+        self.tower_uuid = None
+        uuid = (
+            getattr(settings, 'LOG_AGGREGATOR_TOWER_UUID', None) or
```

Comment:
It's a lot less expensive to just set these once; they're generated at install time anyways.